FabiWare/fabi.h

```diff
@@ -45,8 +45,8 @@
 #define MOUSE_ACCELDELAY   50         // steps to reach mouse move speed
 
 #define MAX_SLOTNAME_LEN      10      // maximum lenght for a slotname
-#define KEYSTRING_BUFFER_LEN 500      // maximum lenght for all string parameters of a slot 
-#define MAX_CMDLEN           200      // maximum lenght of a single AT command
+#define KEYSTRING_BUFFER_LEN 400      // maximum lenght for all string parameters of a slot 
```

Comment:
because of RAM shortage....
sooner or later we will have to change the memory organsiation in the Firmware. This could be improved a lot, if only used buttons / functions would be stored.